### PR TITLE
[Fix: Supports namespaceId.]

### PR DESCRIPTION
**Pull request by koonchen**
#48 
image

### CODE REVIEW

File: example/consul_client/consul_client_test.go [hunk 1]
```diff
@@ -213,98 +213,98 @@ func TestDeregisterServiceInstance(t *testing.T) {
...
-	assert.Nil(t, err)
-
-	instanceDetail2, err := scc.GetServiceInstanceInfo(metaData2)
-	nodes2, ok := instanceDetail2.([]*model.ConsulMetaDataRegister)
-	assert.NotNil(t, nodes2)
-	assert.True(t, ok)
-	assert.Nil(t, err)
-
-	instanceDetail3, err := scc.GetServiceInstanceInfo(metaData3)
-	nodes3, ok := instanceDetail3.([]*model.ConsulMetaDataRegister)
```
Comment (moremind) [inline example/consul_client/consul_client_test.go:302]: why remove this?

**Review by moremind — CHANGES_REQUESTED** [review]

File: example/consul_client/consul_client_test.go (hunk 1 above)
Comment (koonchen) [reply example/consul_client/consul_client_test.go]: The single test content is repeated with `TestEntireConsulFunction`. I am trying to pass ci.

**Comment by koonchen** [comment]
@moremind Can you run ci again? Personal project ci passed.
https://github.com/koonchen/shenyu-client-golang/pull/2

**Comment by lishuo5263** [comment]
pls change your code,about nacos namespaceId(param named like NameSpaceId), the  nacos source code is use "public"
image

**Comment by koonchen** [comment]
I'm not sure if this is correct, PTAL @lishuo5263

File: docker-compose-consul.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@ version: "3.1"
 
 services:
   consul1:
-    image: consul:latest
+    image: consul:1.13.9
```
Comment (moremind) [inline docker-compose-consul.yml:20]: why change the version?

**Review by moremind — CHANGES_REQUESTED** [review]

File: example/http_client/http_client.go [hunk 3]
```diff
@@ -37,38 +37,40 @@ func main() {
...
-		RPCType:     constants.RPCTYPE_HTTP, //require user provide
-		Host:        "127.0.0.1",            //require user provide
-		Port:        "8080",                 //require user provide
+		Protocol:    "http://",                              //require user provide
+		AppName:     "testGoAppName",                        //require user provide
+		ContextPath: "/golang",                              //require user provide
+		RPCType:     constants.RPCTYPE_HTTP,                 //require user provide
+		Host:        "127.0.0.1",                            //require user provide
+		Port:        "8080",                                 //require user provide
+		NamespaceId: "649330b6-c2d7-4edc-be8e-8a54df9eb385", //require user provide
```
Comment (lishuo5263) [inline example/http_client/http_client.go:67]: use "public",not your self config setting.

File: docker-compose-consul.yml (hunk 2 above)
Comment (koonchen) [reply docker-compose-consul.yml:20]: console latest cannot be found

File: example/http_client/http_client.go [hunk 4]
```diff
@@ -63,12 +64,13 @@ func main() {
 		RPCType:     constants.RPCTYPE_HTTP, //require user provide
 		Host:        "127.0.0.1",            //require user provide
 		Port:        "8080",                 //require user provide
+		NamespaceId: "public",               //require user provide
```
Comment (moremind) [inline example/http_client/http_client.go:67]: the namespace is shenyu namespace default value

File: example/http_client/http_client.go [hunk 5]
```diff
@@ -48,12 +48,13 @@ func main() {
 		Enabled:     true,                //require user provide
 		Host:        "127.0.0.1",         //require user provide
 		Port:        "8080",              //require user provide
+		NamespaceId: "public",            //require user provide
```
Comment (moremind) [inline example/http_client/http_client.go:51]: give default value for shenyu namespace id

Comment (moremind) [reply example/http_client/http_client.go:51]: 649330b6-c2d7-4edc-be8e-8a54df9eb385

**Review by moremind — CHANGES_REQUESTED** [review]

**Comment by koonchen** [comment]
@moremind PTAL

**Review by moremind — APPROVED** [review]

**Pull request merged**